hw8/Task_1.ts

```diff
@@ -0,0 +1,16 @@
+// Task 1
+// Выведи все элементы массива в консоль с помощью метода **`forEach`**
+//
+// ```javascript
+//     const fibonacci = [0, 1, 1, 2, 3, 5, 8, 13, 21, 34, 55, 89, 144, 233, 377, 610, 987]
+// ```
+//
+// > Реализуйте решение двумя способами, используя `function declaration` & `arrow function`
+
+const fibonacci: number[] = [0, 1, 1, 2, 3, 5, 8, 13, 21, 34, 55, 89, 144, 233, 377, 610, 987]
+
+fibonacci.forEach((value: number) => {
+    console.log(value)
+});
+
+fibonacci.forEach((value: number) => console.log(value));
```

Comment:
What is the difference between two ways exept new lines in first function?
<img width="730" alt="image" src="https://user-images.githubusercontent.com/47811445/221692121-9c25efe9-1054-4601-85a9-d4a9b8288a83.png">

Reply:
You could not implement function declaration in each tasks, but please do it at leas in 3 tasks and delete redundant arrow functions.

Reply:
I came up with this but have some doubts:

fibonacci.forEach(function (value: number) {
   console.log(value)
});

Please let me know your thoughts on this.

Reply:
As discussed before, this one seems to be correct. So, I added function declarations for Task 1, Task 2 and Task 3.

Reply:
now it looks good, thanks